scavenger-agent-java/src/main/java/com/navercorp/scavenger/javaagent/publishing/GrpcClient.java

```diff
@@ -66,10 +75,11 @@ private void createNewChannelIfShutdown() {
         if (channel == null || channel.isShutdown()) {
             log.fine("[scavenger] creating new grpc channel");
             try {
-                channel = createChannel();
+                int maxMessageSize = System.getProperty(MAX_MESSAGE_SIZE_CONFIG) != null ? maxMessageSize() : DEFAULT_MAX_MESSAGE_SIZE;
```

Comment:
Isn't it better to implement ```System.getProperty(MAX_MESSAGE_SIZE_CONFIG) != null ? maxMessageSize() : DEFAULT_MAX_MESSAGE_SIZE;``` in maxMessageSize()?

Reply:
fixed it in 755bb3d91980ac24c9572a060398d1c60a1b5bd7

Reply:
int maxMessageSize =  maxMessageSize()

Reply:
oh... i missed it... ammend in 6722455b1b873b57577794d5338d447041462e86